CHANGELOG.rst

```diff
@@ -1,3 +1,8 @@
+Version 30.0.0
+==============
+
+- Schema change 30.
+
```

Comment:
Might be worth mentioning your other changes (update workflow, Python versions).